WordPress-VIP/ruleset.xml

```diff
@@ -131,6 +132,11 @@
 	#############################################################################
 	-->
 
+	<!-- Prevent deprecation notice when the sniff is not explicitely included. -->
```

Comment:
Lots of recurring typos: `explicitely` -> `explicitly`.

Reply:
In my defence, this was copy pasted from the previous deprecations

Reply:
Yup. The original spelling mistake is my bad (dyslexic + non-native speaker, so I have an excuse). :sunglasses: